test/PharoLauncherCommonFunctions.sh

```diff
@@ -44,4 +44,12 @@ runLauncherScript() {
 	pushd ..> /dev/null
 	$PHLSCRIPT $@
 	popd> /dev/null
-}
\ No newline at end of file
+}
+
+assertContainsPrinted() {
+    assertContains "Actual: \"$1\", expected: \"$2\". " "$1" "$2"
```

Comment:
shouldn't this be to only print current values if assertion fails?
```bash
assertContains "$1" "$2" || echo "Actual: \"$1\", expected: \"$2\". " "$1" "$2"
```

Reply:
The first parameter of `assertContains` (or `assertNotContains`) is optional message, that is printed only in case, if assertion fails (documentation lacks however description of "message" behavior). Handling you're describe is already there. (I've tested with successful assertion and false assertion).

Reply:
When assertion fails output looks like this: 
`ASSERT:Actual: "blabla", expected: "PhLTestImage". Not found:<PhLTestImage>`
When assert is true, it doesn't print anything.

Reply:
perfect! :)